crates/anemo/src/network/connection_manager.rs

```diff
@@ -791,8 +793,13 @@ mod tests {
         let address = socket.local_addr().unwrap();
         let config = crate::config::EndpointConfig::random("test");
         let endpoint = Arc::new(Endpoint::new(config, socket).unwrap());
+        let manager_config = Config {
+            shutdown_idle_timeout_ms: Some(10_000),
+            ..Default::default()
+        };
+
```

Comment:
Do we need this for tests to pass?

Reply:
Not really, I am just adding some value other than the default which would be infinite here - but can remove.